src/AspNetCoreModuleV2/CommonLib/debugutil.cpp

```diff
@@ -16,11 +16,135 @@
 
 inline HANDLE g_hStandardOutput = INVALID_HANDLE_VALUE;
 inline HANDLE g_logFile = INVALID_HANDLE_VALUE;
+inline HMODULE g_hModule;
 inline SRWLOCK g_logFileLock;
 
+HRESULT
+PrintDebugHeader()
+{
+    DWORD  verHandle = 0;
+    UINT   size      = 0;
+    LPVOID lpBuffer  = NULL;
+
+    WCHAR path[MAX_PATH];
+    RETURN_LAST_ERROR_IF(!GetModuleFileName(g_hModule, path, sizeof(path)));
```

Comment:
Issues with long paths here? We handle long paths in other scenarios.

Reply:
We won't log if ancm.dll is deeper than MAX_PATH, but I'm not sure how much we care.